Description: Print exception traceback when build fails
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/rust-lang/rust/pull/42792
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/bootstrap/bootstrap.py
+++ b/src/bootstrap/bootstrap.py
@@ -684,7 +684,8 @@
             exit_code = e.code
         else:
             exit_code = 1
-            print(e)
+        import traceback
+        traceback.print_exc()
         if not help_triggered:
             print("Build completed unsuccessfully in %s" %
                   format_build_time(time() - start_time))
